scripts/bundleSizeConfig.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
   MIN_SIZE: 814,
-  MAX_SIZE: 858,
+  MAX_SIZE: 881,
```

Comment:
ukrainian bundle size is now the largest 👀 
![image](https://user-images.githubusercontent.com/30599794/85430355-f38c1100-b577-11ea-8608-f53012f5ce06.png)